Project/src/main/resources/templates/editor.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
 <head>
-    <meta charset="UTF-8">
-    <title>Dashboard</title>
```

Comment:
Why are these lines being removed?

Reply:
Part of another PR where theyre being added w/ different names